### PR TITLE
NOD-13 #Comment Connection to the geoplay db + get and post to test c…

### PR DESCRIPTION
**Pull request by MarineCB**

### CODE REVIEW

File: server/init-db.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+			"SELECT tablename FROM pg_tables WHERE schemaname = 'public';"
+		)
+		for (const row of result.rows) {
+			if (row.tablename !== 'spatial_ref_sys') {
+				await postgresStore.client.query(`DROP TABLE IF EXISTS "${row.tablename}" cascade`)
+			}
+		}
+	}
+	await dropEverything()
+	await User.generateTable()
```
Comment (frankfg94) [inline server/init-db.js:20]: Hello, so are we okay saying that for this first API implementation, we don't have any data in the User table and that we must insert it manually? (I currently observe the creation of an empty table, with `User.generateTable()`) 

Thanks

Comment (MarineCB) [reply server/init-db.js:20]: Yes

File: server/models/user.model.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+	/**
+	 * @param {String} username
+	 * @param {String} password
+	 * @returns {Promise<Todo>}
+	 */
+	static async addUser(username, password) {
+		try {
+			const result = await postgresStore.client.query({
+				text: 'INSERT INTO users (username, password) VALUES ($1, $2) RETURNING *',
```
Comment (frankfg94) [inline server/models/user.model.js:31]: Password is not secure

**Review by frankfg94 — CHANGES_REQUESTED** [review]
Password for the user table is not secure

**Review by frankfg94 — APPROVED** [review]
Everything seems good, passwords are encrypted

**Pull request merged**